src/main/java/seedu/anilist/logic/commands/EditCommand.java

```diff
@@ -83,9 +85,11 @@ private static Anime createEditedAnime(Anime animeToEdit, EditAnimeDescriptor ed
         assert animeToEdit != null;
 
         Name updatedName = editAnimeDescriptor.getName().orElse(animeToEdit.getName());
+        Episode episode = animeToEdit.getEpisode();
+        Status status = animeToEdit.getStatus();
```

Comment:
This one dont need optional unlike the rest? Why is it that the rest of them is using optionals?

Reply:
okie coolios